### ISSUE TITLE
selection 2

### ISSUE THREAD

**Issue by Xtopher0s** [Souhardya/WarChild#1]
1.HTTP flood mode simply doesn't work with _https://_. Selection 2 only works with http://site.suffix

2. May I suggest adding a clean exit menu selection. (I would help code it but I am quite busy atm).

3. If using option 2 and use http://site.suffix eventually it will state the "The site is probably down". Instead of exiting program why not continue sending packets?

"This will flood the websites with 100's and 1000's of http header requests and will try to suck all of it's resources.
Select a Target (Please use http/https) :https://www.website.co
[ + ]Starting HTTP Flood !
Traceback (most recent call last):
  File "http_flood.py", line 18253, in <module>
    host = m.group(1)
AttributeError: 'NoneType' object has no attribute 'group'"

**Comment by Souhardya** [comment]
Yeah sure I am working on it thanks for showing concern mate your views are appreciated :D !

**Comment by dnkls** [comment]
Http Flood not work bro, and plz add HEAD/POST/GET methods and we see the power
This code meybe help you https://github.com/dnkls/GoldenEye

**Comment by Souhardya** [comment]
Hello thanks to both of you for participating :100: 
@Xtopher0s bro
1. I have kindly changed the main file issuing a small warning there to follow th instructions to successfully conduct the attacks :) Since I have my exams on head ;_; I will work on the code for sure 

2. Quote "If using option 2 and use http://site.suffix eventually it will state the "The site is probably down". Instead of exiting program why not continue sending packets?" I have currently remove the banner that I was implementing for security measures but since it is not effecting much I have removed the http code callback banners :D ! So it will continue flooding untill you don't command it to stop 

3. About the menu sure I will do it :D 


@dnkls thank you mate for suggestion I was thinking the same and i will implement that too in upcoming versions plus I will improve the cloudflare resolver :D !


Again thanks to both of you for picking up time to use that and report concerning issues mates :100: 
Best of luck  for your projects and I will try to improve it to the fullest 

Regards 
Souhardya

**Comment by Souhardya** [comment]
I am leaving this issue menu open since people may find same problems :D 

@Xtopher0s  Props to you :smile:

**Comment by Lucashuang0802** [comment]
@Souhardya I have the same issue. +1
